cncnet-api/database/migrations/2022_11_25_042019_DeleteSFJLadder.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class DeleteSFJLadder extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $sfjLadder = \App\Ladder::where('abbreviation', 'sfj')->first();
```

Comment:
If I simply delete the SFJ ladder only, will it cascade down and delete all linked data for me? Wondering if  I need a migration file this long to delete the SFJ data

@GrantBartlett